src/freenas/usr/local/lib/middlewared_truenas/plugins/failover_/event.py

```diff
@@ -390,10 +391,10 @@ def carp_master(self, fobj, ifname, vhid, event, user_override, forcetakeover):
                 self.logger.warn('Acquired failover master lock')
                 self.logger.warn('Starting fenced')
                 if not user_override and not fasttrack and not forcetakeover:
-                    error, output = run('LD_LIBRARY_PATH=/usr/local/lib /usr/local/bin/fenced')
+                    error, output = run(f'LD_LIBRARY_PATH=/usr/local/lib /usr/local/bin/fenced -ed {fobj["boot_disks"]}')
```

Comment:
I have doubts this would work given this:

>>> f'{a}'
"['asas', 'asdasd']"
>>> f'-ed {a}'
"-ed ['asas', 'asdasd']"


Should we use {" ".join(fobj["boot_disks"])} ?

Reply:
argsparse does handle a list object but I'll change to join method.

Reply:
What if boot_disks is empty, could that be a possibility in case some parse fails? And if so, would that cause a call error with an empty -ed arg?

Reply:
Yes, it would. argparse exits early if -ed is given with an empty value.

Reply:
Shouldnt we do something about it, then?